MAUI/DataGrid/export-to-pdf.md

```diff
@@ -762,6 +762,10 @@ You can customize the following styles when exporting to PDF as well:
 * GroupCaptionStyle
 * GroupSummaryStyle
 
+### Limitations
+
+* Implicit and explicit styling is not applicable.
```

Comment:
The implicit and explicit styles are not applicable when exporting the DataGrid to PDF.

Reply:
Modified the content